api/source/specification/stig-manager.yaml

```diff
@@ -6200,6 +6662,22 @@ components:
       $ref: '#/components/schemas/String255Nullable'
     IndexDisa:
       $ref: '#/components/schemas/String255'
+    IntegerMinimumZeroNullableNullable:
```

Comment:
IntegerMinimumZeroNullableNullable
 vs 
 IntegerMinimumZeroNullable
 ? 
 IntegerMinimumZeroNullable is not defined as nullable... Do we need both?